draft-ietf-quic-http.md

```diff
@@ -2126,7 +2127,9 @@ Name:
 Description:
 : A brief description of the error code semantics.
 
-The entries in {{iana-error-table}} are registered by this document.
+The entries in {{iana-error-table}} are registered by this document. These
+Error codes were selected from the range that operates on a Specification
+Required policy to avoid collisions with HTTP/2 error codes.
```

Comment:
```suggestion
Required policy (Section 4.6 of {{!RFC8126}}) to avoid collisions
with HTTP/2 error codes.
```

Reply:
I think we can pass on that, as the definition is cited a few paragraphs up.